test/test_c10d.py

```diff
@@ -290,24 +301,15 @@ def test_common_errors(self):
             next(gen)
 
     def test_nominal(self):
-        with tempfile.NamedTemporaryFile() as file:
-            url = 'file://%s?world_size=%d' % (file.name, 2)
+        with tempfile.NamedTemporaryFile(delete=False) as file:
+            url = 'file://%s?world_size=%d' % (file.name, 1)
             gen0 = c10d.rendezvous(url + "&rank=0")
             store0, rank0, size0 = next(gen0)
             self.assertEqual(0, rank0)
-            self.assertEqual(2, size0)
-            gen1 = c10d.rendezvous(url + "&rank=1")
-            store1, rank1, size1 = next(gen1)
-            self.assertEqual(1, rank1)
-            self.assertEqual(2, size1)
+            self.assertEqual(1, size0)
 
-            # Set value on both stores
             store0.set("key0", "value0")
-            store1.set("key1", "value1")
-
-            # Cross check with get
-            self.assertEqual(b"value0", store1.get("key0"))
-            self.assertEqual(b"value1", store0.get("key1"))
+            self.assertEqual(b"value0", store0.get("key0"))
```

Comment:
This test should be restored. It tests the case where you have 2 separate instances of the file store.

Reply:
The rule of thumb is that, if FileStore has a world size of 2, then 2 processes should construct it and destruct it. That's why I removed the store1

Reply:
It doesn't need to be two processes, it can be two instances within the same process. The test ensures that sets on one instance are visible by the other. As well as the proper c10d.rendezvous integration of course.

Reply:
I mean, eventually, I would like the use case only be that, one process opens up one file store. That's it and this is already the case enforced by the front-end API. For this test, it's OK to restore.